### DISCUSSION TITLE
Cosmos Triggers

### DISCUSSION THREAD

**Discussion by daveonplanetearth** [Azure/data-api-builder#1343] [General]
Hi, just tried dab for Cosmos, looks really good. Is it possible to call a pre-trigger from a mutation? ie. I'd like a create/update to be validated before it's committed. Thanks.

**Comment by sajeetharan** [comment]
Thanks for the feedback, hope you liked the experience. Currently we don't support triggers with DAB for Cosmos DB, but definitely we will look into add this feature. Is this for a specific use case scenario?

**Comment by daveonplanetearth** [comment]
Hi, yes I'd like to validate new document field values by looking up data
in another document. I assume the best way to do this is via a pre-trigger.
If the values are not permitted I'd like to rollback the insert.

Thanks.
